### PR TITLE
[TR] Add Performance Points

### PR DESCRIPTION
**Pull request by zeusminus**
Add Turkish locale for `Performance_points` article.

### CODE REVIEW

File: wiki/Performance_points/tr.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Performans puanları
+
+**Performans puanları** (ya da kısaca **pp**) bir oyuncunun osu!'daki kaydettiği ilerlemeyle nispeten doğru orantılı olmayı amaçlayan bir sıralama ölçütüdür.
+
+Bu sistem gelişimin odağının oyun süresinden oyuncunun becerisinin asıl göstergesine kaydırılmasını amaçlamaktadır. Bu, bir [beatmapin](/wiki/Beatmaps) zorluğuna ve bir oyuncunun o beatmapteki performansına dayanan belirli bir skorun hesaplanmasıyla elde edilir.
+
+## Tarihçe
+
+Böyle bir skor sisteminin ilk uygulaması herkese açık olarak Nisan 2012'de duyuruldu ve yalnızca gizemli *'???'* projesi olarak biliniyordu. Bu enigmatik sistem daha sonra gerçek ismine aynı ay içinde kavuştu.
```
Comment (zigizigichan) [inline wiki/Performance_points/tr.md:9]:
```suggestion
Böyle bir skor sisteminin ilk uygulaması herkese açık olarak Nisan 2012'de duyuruldu ve yalnızca gizemli *'???'* projesi olarak biliniyordu. Bu esrarengiz sistem daha sonra gerçek ismine aynı ay içinde kavuştu.
```

File: wiki/Performance_points/tr.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+Piyasaya çıkmasından birkaç ay sonra, 20120722-24 osu! sürümünde, eski [Dereceli](/wiki/Beatmaps#dereceli) skor sisteminin tamamen yerini alacak olan yeni sistem, skor güncellemeleri her 30 dakikada bir hesaplanacak şekilde oyuna resmen uygulandı. Aynı yılın Ağustos ayında sistem gerçek zamanlı güncellenecek şekilde geliştirildi.
+
+*Not: Performans Puanları sisteminin orijinal yapımı olan ppv1 sisteminin de bir değişiklik kaydı vardır. Bunu, ilgili [forum konusundan](https://osu.ppy.sh/community/forums/topics/92185) görüntüleyebilirsiniz.*
+
+Bir yılı aşkın bir süre boyunca bu kapasitede devam etti, ta ki *osu!tp* puanlama ölçütünün yaratıcısı [Tom94](https://osu.ppy.sh/users/1857058) [osu! team](/wiki/People/The_Team)'e katılana ve kendi dizaynını mevcut sisteme uygulayana kadar. Sonucunda ortaya çıkan sistem *ppv2* olarak adlandırıldı, ve 27 Ocak 2014 tarihinde güncel sürüme eklendi. Böylece eski sistem de *[ppv1](/wiki/Performance_points/ppv1)* olarak yeniden adlandırıldı.
+
+ppv2 şu anda aktif olarak, kendi [değişiklik kaydına](https://osu.ppy.sh/p/changelog?category=pp) zamanla eklenen güncellemelerle hizmet vermektedir.
```
Comment (zigizigichan) [inline wiki/Performance_points/tr.md:19]:
```suggestion
ppv2 şu anda aktif olarak, kendi [değişiklik kaydına](https://osu.ppy.sh/p/changelog?category=pp) zamanla eklenen güncellemelerle hizmet vermeye devam etmektedir.
```

File: wiki/Performance_points/tr.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Skorlar daha sonrasında kullanıcının yalnızca en iyi skorlarının çoğunlukla genel performans puanı sıralamasına sayılmasını sağlamak için birbirlerine karşı "ağırlıklandırılır". Bu, [*ağırlık sistemi*](#ağırlık-sistemi) olarak bilinir, amacı ise oyuncunun diğer skorlarına göre kazanılan pp miktarını azaltarak kolay beatmaplerde sürekli bir şekilde düşük pp skorları kasılmasını önlemektir.
+
+*Not: Skor kaydettiğiniz Dereceli map sayısına göre ufak bir miktar bonus pp kazanabilirsiniz.*
+
+### Ağırlık sistemi
+
+Ağırlık sistemi, bir oyundan elde edilen toplam performas puanının hesaplanmasından sonra kullanılan basit bir formüldür. Formül, oyuncunun en iyi skorları arasındaki sıralamasına göre kazanılan pp miktarını azaltmak için kullanılır. Bahsi geçen formül aşağıdaki gibidir:
```
Comment (zigizigichan) [inline wiki/Performance_points/tr.md:33]:
```suggestion
Ağırlık sistemi, bir oyundan elde edilen toplam performans puanının hesaplanmasından sonra kullanılan basit bir formüldür. Formül, oyuncunun en iyi skorları arasındaki sıralamasına göre kazanılan pp miktarını azaltmak için kullanılır. Bahsi geçen formül aşağıdaki gibidir:
```

File: wiki/Performance_points/tr.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+**Yukarıda belirtildiği üzere, eski skorlar zamanla kendi toplam değerlerinin yüzde birinden daha az bir miktarda ağırlıklandırılacaktır. Bu, siz geliştikçe eninde sonunda toplam skorunuzu neredeyse hiç etkilemeyeceği anlamına geliyor.**
+
+Lakin o noktada çok daha etkileyici skorlar kaydetmiş olacaksınız, yani elde ettiğiniz daha yüksek skorlar eskilerini geride bıraktığında toplam pp miktarınız da artacak.
+
+### Neden yeni bir skor kaydettiğim halde pp kaybettim?
+
+**Öncekinden daha düşük isabet oranıyla daha yüksek bir komboya sahip veya modlarla oynanmış bir skor kaydetmeniz dolayısıyla kimi zaman pp kaybedebilirsiniz.**
+
+Toplam skor halen daha bireysel map sıralamaları için önemlidir ve bu daha düşük bir isabet oranına sahip olsa da skor miktarı daha yüksek veya mod kullanılan bir skorun "daha iyi" bir sonuç çıkarmış olsa da yine de pp kaybetmenize sebep olduğu durumlar oluşturabilir.
```
Comment (zigizigichan) [inline wiki/Performance_points/tr.md:116]:
```suggestion
Toplam skor halen daha bireysel map sıralamaları için önemlidir ve bu daha düşük bir isabet oranına sahip olsa da skor miktarı daha yüksek veya mod kullanılan bir skor "daha iyi" bir sonuç çıkarmış olsa da yine de pp kaybetmenize sebep olduğu durumlar oluşturabilir.
```

**Review by zigizigichan — CHANGES_REQUESTED** [review]
That's all I can suggest. I pointed out a few incoherencies and redacted a miswritten word. Looks good.

**Review by TicClick — APPROVED** [review]

**Pull request merged**